python/src/GraphImplementation_doc.i.in

```diff
@@ -299,6 +299,49 @@ OT_Graph_IsValidLegendPosition_doc
 
 // ---------------------------------------------------------------------
 
+%define OT_Graph_setLegendCorner_doc
+"Accessor to the legend's corner.
+
+Parameters
+----------
+corner : sequence of float or size 2
+    Relative coordinates (in [0,1]x[0,1]) of the legend in the graph bounding box.
+    The position of the legend relatively to this point will depend on the orientation
+    given by the :meth:`getLegendPosition` value.
+
+Examples
+--------
+>>> import openturns as ot
+>>> position = 'topleft'  # the legend's topleft corner with be set below
+>>> graph = ot.Graph('Some curves', 'x1', 'x2', True, position, 1.0, 0)
+>>> graph.setLegendCorner([1.0, 1.0])  # legend will begin in top right corner
+
+Notes
+-----
+This allows to place the legend outside the graph.
```

Comment:
D'après "_Pratique de l'anglais de A à Z_", Hatier, 1983, page 178 le verbe permettre peut avoir 2 sens:
- donner la permission : "allow"
- rendre possible : "enable" ou "make it possible"

Ici (et comme dans la plupart de nos doc API), c'est le second sens que l'on souhaite.
```suggestion
This makes it possible to place the legend outside the graph.
```